### PR TITLE
Handleconversion calling

### LINKED ISSUE #5: Change handleConversion() to receive two parameters (source and target)
As of now the function handleConversion() is receiving the type of conversion as a parameter:
```html
        <input id="decToBin" type="button" value="binary" onclick="handleConversion('decToBin')" />
        <input id="decToHex" type="button" value="hexadecimal" onclick="handleConversion('decToHex')" />
        <input id="decToOct" type="button" value="octal" onclick="handleConversion('decToOct')" />
```

and is a clearer way to manage that function separating the input and the output number type, changing:
`handleConversion('decToBin')`
to:
`handleConversion('dec', 'bin')`

### PR DESCRIPTION
**Pull request by GonzoVM**
I changed the handleConversion() calling form one parameter (kind of conversion) to two parameters (source format and target format) so I can manage better:
```html
        <!-- from: -->
        <input id="binToDec" type="button" value="decimal" onclick="handleConversion('binToDec')" />
        <!--- to: -->
        <input id="binToDec" type="button" value="decimal" onclick="handleConversion('binary', 'decimal')" />
```

This change improves the code because erase some redundancy on the source format validation calling, and there is no need to create news variables (inputNumber and outputFormat) to manage this information because the function receive it from the calling.

### CODE REVIEW

File: src/display_output.js [hunk 1]
```diff
@@ -1,79 +1,71 @@
...
-      displayOutput(octToDec(num), outputFormat);
-      break;
-    case "octToHex":
-      inputNumber = "octal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(octToHex(num), outputFormat);
-      break;
-    default:
-      return alert("whta did you do?");
+  if ((srcFormat == "binary")) {
```
Comment (joaquimgamero) [inline src/display_output.js]:
```suggestion
  if (srcFormat === "binary") {
```

File: src/display_output.js [hunk 2]
```diff
@@ -1,79 +1,71 @@
...
+      case "hexadecimal":
+        displayOutput(binToHex(num), targetFormat);
+        break;
+      case "octal":
+        displayOutput(binToOct(num), targetFormat);
+        break;
+      default:
+        return alert("what did you do?");
+    }
+  } else if ((srcFormat == "decimal")) {
```
Comment (joaquimgamero) [inline src/display_output.js]:
```suggestion
  } else if (srcFormat === "decimal") {
```

File: src/display_output.js [hunk 3]
```diff
@@ -1,79 +1,71 @@
...
+      case "hexadecimal":
+        displayOutput(decToHex(num), targetFormat);
+        break;
+      case "octal":
+        displayOutput(decToOct(num), targetFormat);
+        break;
+      default:
+        return alert("What did you do?");
+    }
+  } else if ((srcFormat == "hexadecimal")) {
```
Comment (joaquimgamero) [inline src/display_output.js]:
```suggestion
  } else if (srcFormat === "hexadecimal") {
```

File: src/display_output.js [hunk 4]
```diff
@@ -1,79 +1,71 @@
...
+      case "decimal":
+        displayOutput(hexToDec(num), targetFormat);
+        break;
+      case "octal":
+        displayOutput(hexToOct(num), targetFormat);
+        break;
+      default:
+        return alert("What did you do?");
+    }
+  } else if ((srcFormat == "octal")) {
```
Comment (joaquimgamero) [inline src/display_output.js]:
```suggestion
  } else if (srcFormat === "octal") {
```

**Review by joaquimgamero — CHANGES_REQUESTED** [review]
Good job! Two things, no need to wrap if statements in double parentheses, and always try to use strict equals (`===`) rather than loose equal (`==`).

File: src/display_output.js (hunk 1 above)
Comment (GonzoVM) [reply src/display_output.js]: The double parentheses was something that added prettier when i formatted the document, not sure why.
¿What improves using strict equals on this case? I mean, I'm comparing a string with a string and I don't see the data type could induce an error. Btw, I changed as you suggested.
https://github.com/GonzoVM/bin2dec/commit/2a28d4dc0280f4c01b24b0333fff72d71f7d5d15

File: src/display_output.js (hunk 2 above)
Comment (GonzoVM) [reply src/display_output.js]: https://github.com/GonzoVM/bin2dec/commit/2a28d4dc0280f4c01b24b0333fff72d71f7d5d15

File: src/display_output.js (hunk 3 above)
Comment (GonzoVM) [reply src/display_output.js]: https://github.com/GonzoVM/bin2dec/commit/2a28d4dc0280f4c01b24b0333fff72d71f7d5d15

File: src/display_output.js (hunk 4 above)
Comment (GonzoVM) [reply src/display_output.js]: https://github.com/GonzoVM/bin2dec/commit/2a28d4dc0280f4c01b24b0333fff72d71f7d5d15

**Comment by GonzoVM** [comment]
> Good job! Two things, no need to wrap if statements in double parentheses, and always try to use strict equals (`===`) rather than loose equal (`==`).

The double parentheses was something that added prettier when i formatted the document, not sure why.
¿What improves using strict equals on this case? I mean, I'm comparing a string with a string and I don't see the data type could induce an error. Btw, I changed as you suggested.
https://github.com/GonzoVM/bin2dec/commit/2a28d4dc0280f4c01b24b0333fff72d71f7d5d15

**Pull request merged**